images/capi/ansible/roles/sysprep/tasks/main.yml

```
@@ -130,6 +130,7 @@
   loop:
   - { path: /var/log, state: absent }
   - { path: /var/log, state: directory }
+  - { path: /var/log/sa, state: directory, mode: "0755" }
```

mode looks to be hardcoded above
```suggestion
  - { path: /var/log/sa, state: directory }
```